hrm-service/service-tests/contact-job/jobTypes/retrieve-transcript.test.ts

```diff
@@ -394,21 +409,21 @@ describe('complete retrieve-transcript job type', () => {
       );
 
       const err = new Error('something went wrong');
-      const errToObject = Object.fromEntries(Object.getOwnPropertyNames(err).map(n => [n, err[n]]));
+      const errMessage = err instanceof Error ? err.message : String(err);
```

Comment:
Transcripts only throws a stringifiied payload so I updated the backend to only accept a string. This could be undone if there is a reason we want objects in the future?